src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -2740,7 +2733,7 @@ public enum HeapIndex
     }
     public sealed partial class MetadataAggregator
     {
-        public MetadataAggregator(System.Collections.Generic.IReadOnlyList<int> baseTableRowCounts, System.Collections.Generic.IReadOnlyList<int> baseHeapSizes, System.Collections.Generic.IReadOnlyList<System.Reflection.Metadata.MetadataReader> deltaReaders) { }
+        public MetadataAggregator(System.Collections.Generic.IReadOnlyList<int>? baseTableRowCounts, System.Collections.Generic.IReadOnlyList<int>? baseHeapSizes, System.Collections.Generic.IReadOnlyList<System.Reflection.Metadata.MetadataReader>? deltaReaders) { }
         public MetadataAggregator(System.Reflection.Metadata.MetadataReader baseReader, System.Collections.Generic.IReadOnlyList<System.Reflection.Metadata.MetadataReader> deltaReaders) { }
         public System.Reflection.Metadata.Handle GetGenerationHandle(System.Reflection.Metadata.Handle handle, out int generation) { throw null; }
     }
```

Comment:
Below row 2746 `object value` is nullable

Reply:
Good catch, fixed.